.github/workflows/test.yaml

```diff
@@ -0,0 +1,16 @@
+on: [push]
+name: Test action
+
+jobs:
+  push_artifacts_action:
+    runs-on: ubuntu-latest
+    name: A job that push file on artifacts
+    steps:
+      - uses: actions/checkout@v1
+      - id: Test-action
+        uses: ./
+        with:
+          artifacts_url: https://artifacts-devl.scality.net
+          username: ${{ secrets.ARTIFACTS_USER }}
+          password: ${{ secrets.ARTIFACTS_PASSWORD }}
+          sources: action.yml README.md
```

Comment:
We need a bit more testing scenarios.
* To start, every file you upload, please download them back and verify there integrity (sha256 sum or md5)
* Create a new scenario with multiple files to be uploaded (100 per example)
* Create a new scenario with a big file (around 1Gb) to ensure we are able to upload and retrieve them.

And the other scenarios will come in the future